### ISSUE TITLE
Unmocked URIs on non-standard ports block forever

### ISSUE THREAD

**Issue by markstory** [gabrielfalcao/HTTPretty#187]
I ran into an issue where I need to mock multiple webservices, but not mock Elasticsearch (my datastore). I've not had any luck in making httpretty allow requests to port 9200 through. An short script showing my issue is:

```python
import httpretty
import requests

httpretty.enable()
httpretty.register_uri(httpretty.GET, 'http://localhost/test', body="I am a teapot")

print 'Getting a mocked uri'
res = requests.get('http://localhost/test')
print res

print 'Getting an unmocked uri on standard port'
res = requests.get('http://localhost')
print res

print 'Getting an unmocked uri'
res = requests.get('http://localhost:9200/')
print res
```

I've not explored the test cases enough to figure out how to add a new one, but if I can help in any other way please let me know. I'm using Python 2.7.3 if that matters.

Edit: Add version info.

**Comment by gabrielfalcao** [comment]
This might have been fixed by @frankamp on #216

**Comment by anentropic** [comment]
I don't think so: #65
